src/test/java/net/snowflake/client/jdbc/BindUploaderLatestIT.java

```diff
@@ -18,9 +17,11 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.Arrays;
+import java.util.Map;
```

Comment:
Some of these tests are testing the ability to create a csv file on local disk, which does not happen anymore. The old upload function is now called uploadDeprecated. When testing csv file generation, uploadDeprecated is used instead.

Reply:
Why do we still leave these tests here? Why not convert all of them to use the new upload method?

Reply:
Old tests are now deleted. I added a couple more to test size of input stream.